office/dspdfviewer/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           qt5 1.0
+
+github.setup        dannyedel dspdfviewer 1.15.1 v
+categories          office
+platforms           darwin
+license             GPL-2+
+maintainers         {@ryanakca debian.org:rak} \
+                    openmaintainer
+description         dual-screen LaTeX beamer presentation viewer
+long_description    Provides a dual-screen presentation viewer for slides \
+                    produced with the beamer LaTeX package. It takes slides \
+                    produced with the "show notes on second screen" beamer \
+                    option, splits them, and shows the slides in one window \
+                    and the notes in a second window. The second window also \
+                    includes a talk timer, a preview of upcoming slides, and \
+                    other features.
+
+checksums           rmd160  82b190c46afe5fdf1feb5d121cf9a25a53da9612 \
+                    sha256  38a78b9d04ccc6eccea67857032baab06fc85e142c1c442253efe1466841dfd8 \
+                    size    165198
+
+qt5.depends_component \
+                    qttools
+
+depends_build-append \
+                    port:cmake \
```

Comment:
The cmake portgroup will have already added cmake as a dependency. So it should be removed here.